pubspec.lock

```diff
@@ -43,13 +43,27 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "1.15.0"
+  crypto:
```

Comment:
Do we use it or no?

Reply:
We need it when calculate md5. Now we dont use it. remove this dependency.